tests/foreman/api/test_computeresource_libvirt.py

```diff
@@ -20,132 +20,115 @@
 
 
-@pytest.mark.tier1
-@pytest.mark.parametrize('name', **parametrized(valid_data_list()))
-def test_positive_create_with_name(setup, name):
-    """Create compute resources with different names
+@pytest.mark.e2e
+def test_positive_crud_libvirt_cr(module_target_sat, module_org, module_location):
+    """CRUD compute resource libvirt
 
-    :id: 1e545c56-2f53-44c1-a17e-38c83f8fe0c1
+    :id: 1e545c56-2f53-44c1-a17e-38c83f8fe0c2
 
     :expectedresults: Compute resources are created with expected names
 
     :CaseImportance: Critical
 
     :CaseLevel: Component
-
-    :parametrized: yes
     """
-    compresource = entities.LibvirtComputeResource(
-        location=[setup.loc],
+    name = gen_string('alphanumeric')
+    description = gen_string('alphanumeric')
+    display_type = 'spice'
+    libvirt_url = 'qemu+tcp://libvirt.example.com:16509/system'
+
+    cr = module_target_sat.api.LibvirtComputeResource(
         name=name,
-        organization=[setup.org],
-        url=setup.current_libvirt_url,
+        description=description,
+        provider=FOREMAN_PROVIDERS['libvirt'],
+        display_type=display_type,
+        organization=[module_org],
+        location=[module_location],
+        url=libvirt_url,
     ).create()
-    assert compresource.name == name
+    assert cr.name == name
+    assert cr.description == description
+    assert cr.provider == FOREMAN_PROVIDERS['libvirt']
+    assert cr.display_type == display_type
+    assert cr.url == libvirt_url
+
+    # Update
+    new_name = gen_string('alphanumeric')
+    new_description = gen_string('alphanumeric')
+    new_display_type = 'vnc'
+    new_org = module_target_sat.api.Organization().create()
+    new_loc = module_target_sat.api.Location(organization=[new_org]).create()
+    cr.name = new_name
+    cr.description = new_description
+    cr.display_type = new_display_type
+    cr.url = LIBVIRT_URL
+    cr.organization = [new_org]
+    cr.location = [new_loc]
+    cr.update(['name', 'description', 'display_type', 'url', 'organization', 'location'])
+
+    # READ
+    updated_cr = module_target_sat.api.LibvirtComputeResource(id=cr.id).read()
+    assert updated_cr.name == new_name
+    assert updated_cr.description == new_description
+    assert updated_cr.display_type == new_display_type
+    assert updated_cr.url == LIBVIRT_URL
+    assert updated_cr.organization[0].id == new_org.id
+    assert updated_cr.location[0].id == new_loc.id
+    # DELETE
+    updated_cr.delete()
+    assert not module_target_sat.api.LibvirtComputeResource().search(
+        query={'search': f'name={new_name}'}
+    )
 
 
 @pytest.mark.tier1
-@pytest.mark.parametrize('description', **parametrized(valid_data_list()))
-def test_positive_create_with_description(setup, description):
-    """Create compute resources with different descriptions
+@pytest.mark.parametrize('name', **parametrized(valid_data_list()))
+def test_positive_create_with_name_description(
```

Comment:
Isn't this already covered in `test_positive_crud_libvirt_cr`

Reply:
test_positive_crud_libvirt_cr covers only alpha name, and this test exercises CR creation with all valid names from valid_data_list